sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -0,0 +1,741 @@
+## API Report File for "@azure/ai-form-recognizer"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { AbortSignalLike } from '@azure/core-http';
+import * as coreHttp from '@azure/core-http';
+import { HttpRequestBody } from '@azure/core-http';
+import { OperationOptions } from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { PipelineOptions } from '@azure/core-http';
+import { Poller } from '@azure/core-lro';
+import { PollerLike } from '@azure/core-lro';
+import { PollOperationState } from '@azure/core-lro';
+import { ServiceClientCredentials } from '@azure/core-http';
+import { TokenCredential } from '@azure/identity';
+import { WebResource } from '@azure/core-http';
+
+// @public (undocumented)
+export type AnalyzeFormResult = Omit<AnalyzeResult, "documentResults">;
+
+// @public
+export interface AnalyzeLayoutAsyncHeaders {
+    operationLocation: string;
+}
+
+// @public
+export type AnalyzeLayoutAsyncResponseModel = AnalyzeLayoutAsyncHeaders & {
+    _response: coreHttp.HttpResponse & {
+        parsedHeaders: AnalyzeLayoutAsyncHeaders;
+    };
+};
+
+// @public (undocumented)
+export type AnalyzeLayoutOperationResult = Omit<AnalyzeOperationResultModel, 'analyzeResult'> & {
+    analyzeResult?: AnalyzeLayoutResult;
+};
+
+// @public (undocumented)
+export type AnalyzeLayoutResult = Omit<AnalyzeResult, 'documentResults'>;
+
+// @public
+export type AnalyzeLayoutResultResponse = AnalyzeLayoutOperationResult & {
+    _response: coreHttp.HttpResponse & {
+        bodyAsText: string;
+        parsedBody: AnalyzeOperationResultModel;
+    };
+};
+
+// @public
+export interface AnalyzeOperationResultModel {
+    analyzeResult?: AnalyzeResultModel;
+    createdDateTime: Date;
+    lastUpdatedDateTime: Date;
+    status: OperationStatus;
+}
+
+// @public (undocumented)
+export type AnalyzeOptions = ExtractReceiptOptions | ExtractLayoutOptions | ExtractCustomFormOptions;
+
+// Warning: (ae-forgotten-export) The symbol "ICanHazStatus" needs to be exported by the entry point index.d.ts
+//
+// @public
+export type AnalyzePollerClient<T extends ICanHazStatus> = {
```

Comment:
generic poller is only for easier implementation/code sharing.  If separate copy of the pollers are better for public api surface we could make three almost identical pollers.